webapp/locales/en.json

```
@@ -306,9 +306,9 @@
   "editor": {
     "embed": {
       "always_allow": "Always allow embedded content by third party providers (this setting can be changed any time)",
-      "data_privacy_info": "Your data has not yet been shared with any third party providers. If you proceed to watch this video the following provider will likely collect user data:",
+      "data_privacy_info": "Your data has not yet been shared with any third party providers. If you proceed to watch this content the following provider will likely collect user data:",
```

```suggestion
      "data_privacy_info": "Your data has not yet been shared with any third party providers. If you proceed to play this content the following provider will likely collect user data:",
```

We've overseen to replace the word `watch` with `play` @Thiruppathi , because it shall be working for sound only as well.

---

Haa.. good point. Will commit this suggestion.